src/main/java/chungus/Chungus.java

```diff
@@ -0,0 +1,98 @@
+package chungus;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import chungus.textui.TextUi;
+
+/**
+ * A task management app.
+ */
+public class Chungus {
+    private NonBlockingUi ui;
+    private TaskList tasks;
+    private Storage db;
+    private AtomicBoolean isRunning;
+
+    public static final String DEFAULT_DB_PATH = System.getProperty("user.dir") + "/chungus.db";
```

Comment:
Perhaps the static attributes should be above the private final attributes according to coding standards

Reply:
In which page did you find this? I wasn't able to find it at https://se-education.org/guides/conventions/java/intermediate.html

Reply:
Tbh idk someone suggested that to me too.

Reply:
lol